hs_core/templatetags/hydroshare_tags.py

```diff
@@ -131,6 +131,27 @@ def best_name(content):
     return content
 
 
+@register.filter
+def name_without_commas(name):
+    """
+    Takes a name formatted as "[LastNames], [FirstNames]"
+    and returns it formatted as "[FirstNames] [LastNames]".
+    If a name without commas is passed it is returned unchanged.
+    """
+
+    if "," in name:
+        name_parts = name.split(",")
+
+        if len(name_parts) == 1:
+            return name_parts[0]
```

Comment:
This will never execute and the whole block should be functionally equivalent to the logic above.